inc/functions/cloudflare.php

```diff
@@ -41,15 +41,41 @@ function rocket_is_api_keys_valid_cloudflare( $cf_email, $cf_api_key, $cf_zone_i
 
 	try {
 		$cf_api_instance = new Cloudflare\Api( $cf_email, $cf_api_key );
-		$cf_user         = $cf_api_instance->get('user/');
-		$cf_zone         = $cf_api_instance->get('zones/'.$cf_zone_id);
+		$cf_user         = $cf_api_instance->get( 'user/' );
+		$cf_zone         = $cf_api_instance->get( 'zones/' . $cf_zone_id );
 
 		if ( ! isset( $cf_zone->success ) || empty( $cf_zone->success ) ) {
 			return new WP_Error( 'cloudflare_invalid_auth', $cf_zone->errors[0]->message );
 		}
 
 		if ( true === $cf_zone->success ) {
-		    return true;
+			$zone_found = false;
+
+			foreach ( $cf_zone->result as $zone ) {
+				if ( get_site_url() === $zone->name ) {
+					$zone_found = true;
+				}
+			}
+
+			if ( ! $zone_found ) {
+				$msg = sprintf(
+					// translators: %s = WP Rocket plugin name.
+					__( 'It looks that you domain is not provisioned with Cloudflare.', 'rocket' ),
+					WP_ROCKET_PLUGIN_NAME
+				);
```

Comment:
You use `sprintf`, but there is no corresponding token `%s` in the string. It could be:

`__( '%s: It looks like your domain is not provisioned with Cloudflare.', 'rocket' )`

Reply:
@crystinutzaa There is a minor typo in `you domain` in the original message. (It's fixed in what Remy suggested)

I have asked @webtrainingwheels for feedback on the message. Please take a look at the doc when you make this change - https://docs.google.com/spreadsheets/d/1v0MJi-m7qtGOjqctA946JL1GdZqXNM1svEACxQcsago/edit#gid=0

Reply:
@crystinutzaa To make this easier, this is what Lucy suggested: 

```
**WP Rocket**: It looks like your domain is not set up on Cloudflare. Read the documentation for further guidance.
```